### DISCUSSION TITLE
I will combine my meeting Card with my calendar & i have 5 error on my calendar

### DISCUSSION THREAD

**Discussion by Nassir-Al** [neuefische/questions#245] [Web] [answered]
Hey Guys, i will combine my createt meeting card with my calendar but i dont know how to do that. Do you have a idea?

here is my code:
https://github.com/Nassir-Al/capstone-project

here is the libary i used for my calendar:
https://github.com/wojtekmaj/react-calendar

Ziel:

<img width="343" alt="Bildschirmfoto 2022-10-12 um 00 09 01" src="https://user-images.githubusercontent.com/109968270/195207852-8992213a-246f-4eb6-b69c-c3f3fbb2790f.png">

Overview:

<img width="514" alt="Bildschirmfoto 2022-10-11 um 21 44 15" src="https://user-images.githubusercontent.com/109968270/195207999-abcde1ff-3651-49ac-81bf-baae30fe8f48.png">

<img width="343" alt="Bildschirmfoto 2022-10-12 um 00 09 01" src="https://user-images.githubusercontent.com/109968270/195208039-70541e79-9043-403b-92e7-e7d68c125ebf.png">

**Comment by mbosselmann** [comment, marked as answer]
Hi Nassir,
Julia and I checked your code and came to the following solution: 
- You need to move the `dateState` and the `changeDate` function from your `Calendar` component to your `index.js` (cue: uplifting state). 
- The next step is to change your `handleAddToDo` function and add a new key/ value pair: `date: dateState`. 
 
The code in your `index.js` should look like this:

```js
export default function Home() {
	const todos = useStore(state => state.todos);
	const setTodos = useStore(state => state.setTodos);
	const [dateState, setDateState] = useState(new Date());
	const [searchQuery, setSearchQuery] = useState('');

	const handleDeleteTodo = id => {
		setTodos(todos.filter(todo => todo.id !== id));
	};

	const handleAddtodo = event => {
		if (event.key === 'Enter') {
			setTodos([
				...todos,
				{id: uuid(), title: event.target.value, done: false, date: dateState},
			]);
			event.target.value = '';
		}
	};

	const changeDate = event => {
		setDateState(event);
	};
```
Currently no date is displayed inside your `todo` items. Therefore, check via `console.log` if your `todos` now have a `date` `key` with the right `date` `string`.

I hope this solution is helpful. :)

**Comment by Nassir-Al** [comment]
Thanks guys, i will keep you updated!
